tsconfig.json

```
@@ -14,9 +14,10 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
-    "baseUrl": ".",
+    "baseUrl": "./src",
     "paths": {
       "@/*": ["./*"],
```

I'd like to maybe rename this to
```suggestion
      "client/*": ["./*"],
```

for consistency but that can be a future PR